### PR TITLE
Add ServiceAsync

### PR DESCRIPTION
**Pull request by MrDragar**

### CODE REVIEW

**Comment by issamansur** [comment]
@MrDragar 
Hi, thank u for your PR. I will look at the changes and make a decision. I really don't know if this asynchrony is necessary for the `Service`, but in any case, thank you very much!

**Comment by MrDragar** [comment]
I just forked you're library for my Telegram bot. Thanks for vkpymusic

**Comment by issamansur** [comment]
glad to help) Question: Did you need an asynchronous version of the `Service` for the bot to work?

**Comment by MrDragar** [comment]
I could use a synchronous version, but I don't want to deal with multithreading problems (I use Aiogram and Tortoise ORM)

**Comment by MrDragar** [comment]
Last 2 commits were made for myself. Don't apply them please

File: vkpymusic/Service.py [hunk 1]
```diff
@@ -338,7 +338,7 @@ def search_songs_by_text(
         logger.info(f'Request by text: "{text}" в количестве {count}')
 
         try:
-            response = self.__search(text, count)
+            response = self.__search(text, count, offset)
```
Comment (issamansur) [inline vkpymusic/Service.py:341]: Yeah, it was my mistake) Thank u)

File: examples/search_and_save_music_async_example.py
Comment (issamansur) [inline examples/search_and_save_music_async_example.py:1]: Good example!

File: vkpymusic/ServiceAsync.py
Comment (issamansur) [inline vkpymusic/ServiceAsync.py:1]: Good job, but I returned output params to old version)

Changes:
- Change types from `turple[list[Song], int]` to `list[Song]`
- Delete output var `count` from `__response_to_songs(...)`

**Review by issamansur — APPROVED** [review]
Thanks for ur work! Good job :3

**Comment by issamansur** [comment]
Thank u again, but next time please:
- **create new branch** in forked repository with name associated with issue. For example, `add-service-async`)
- if u do some mistake in commit, use `amend` for apply this changes in last commit, don't in new)
- if u want add some changes for yourself u can create another branch)

P. S. this is just advices)

**Pull request merged**